### ISSUE TITLE
In ios RNZendesk undefined

### ISSUE THREAD

**Issue by romreed** [dcvz/react-native-zendesk#17]
I install react-native-zendesk like in readme wrote and try RNZendesk.initialize but RNZendesk is undefined. I found 
```
var RNZendesk = react_native_1.NativeModules.RNZendesk;
// MARK: - Initialization
function initialize(config) {
    RNZendesk.initialize(config);
}
```
return RNZendesk undefined. What i made wrong ?

"react-native": "0.61.5",
"react-native-zendesk": "^2.1.0"
xcode Version 11.2.1 (11B500)

**Comment by MariusCatanoiu** [comment]
> I install react-native-zendesk like in readme wrote and try RNZendesk.initialize but RNZendesk is undefined. I found
> 
> ```
> var RNZendesk = react_native_1.NativeModules.RNZendesk;
> // MARK: - Initialization
> function initialize(config) {
>     RNZendesk.initialize(config);
> }
> ```
> 
> return RNZendesk undefined. What i made wrong ?
> 
> "react-native": "0.61.5",
> "react-native-zendesk": "^2.1.0"
> xcode Version 11.2.1 (11B500)

you run pod install?

**Comment by romreed** [comment]
@MariusCatanoiu yes. i think problem was in ```platform :ios, '9.0'```. Min version now 9.3

**Comment by MariusCatanoiu** [comment]
@romreed  I have the same error now. But my version is 10.0. Is working for you?

**Comment by romreed** [comment]
@MariusCatanoiu after update platform to 9.3  in pod file issue is gone.

**Comment by romreed** [comment]
@MariusCatanoiu if you need chat try react-native-zendesk-ichat

**Comment by MariusCatanoiu** [comment]
@romreed not really. but Thanks!
